cmd/sonobuoy/app/kubeconfig.go

```diff
@@ -37,6 +37,14 @@ func (c *Kubeconfig) Set(str string) error {
 
 // Get returns a rest Config, possibly based on a provided config
 func (c *Kubeconfig) Get() (*rest.Config, error) {
+
+	// detect if in-cluster configuration makes sense
+	if len(c.ExplicitPath) > 0 {
```

Comment:
whoops this is backwards

Reply:
probably easier just to compare to empty string.